src/interfaces/validator/index.js

```diff
@@ -22,7 +22,7 @@ import {LOG_ITEM_TYPE} from '@natlibfi/melinda-rest-api-commons/dist/constants';
 //const debug = createDebugLogger('@natlibfi/melinda-rest-api-validator:validator');
 //const debugData = debug.extend('data');
 
-export default async function ({preValidationFixOptions, postMergeFixOptions, preImportFixOptions, sruUrl, matchOptionsList, mongoLogOperator, recordType, stopWhenFound}) {
+export default async function ({preValidationFixOptions, postMergeFixOptions, preImportFixOptions, sruUrl, matchOptionsList, mongoLogOperator, recordType, stopWhenFound, acceptZeroWithMaxCandidates}) {
```

Comment:
Long parametter lists would be clearer in stagged style
```js
export default async function ({
  preValidationFixOptions,
  postMergeFixOptions,
  preImportFixOptions,
  sruUrl,
  matchOptionsList,
  mongoLogOperator,
  recordType,
  stopWhenFound,
  acceptZeroWithMaxCandidates
}) {
```

Reply:
Very true, will fix.